doc/releases/luminous.rst

```diff
@@ -1,3 +1,157 @@
+v12.2.11 Luminous
+=================
+
+This is the eleventh bug fix release of the Luminous v12.2.x long term stable
+release series. We recommend that all users upgrade to this release. Please note
+the following precautions while upgrading.
+
+Notable Changes
+---------------
+
+* A previous release, v12.2.9, introduced the PG hard-limit patches which were
+  found to cause an issue in certain upgrade scenarios, and 12.2.10 release was
+  expedited to revert those patches. If you already successfully installed
+  v12.2.9, then you can upgrade to 12.2.11 directly. 12.2.10 users need no
+  special precaution.
```

Comment:
This is confusing.  12.2.10 need no precaution.  but 12.2.9 users can also upgrade directly.  isn't that everyone?

Reply:
@theanalyst @liewegas I don't think we need to mention any special precaution. 
It is important to mention that users need to set the flag, in order to make use of this feature. Also, setting this flag will only succeed once the cluster has completely upgraded to 12.2.11. Does this make it clear?

Reply:
So everyone after upgrade must set this flag, regardless of wherever they're upgrading from, is that correct?

Reply:
They are not required to, but they "must" only if they want to limit the length of their pg logs.